src/components/ActionButton/ActionButton.scss

```diff
@@ -0,0 +1,60 @@
+@import '../../sass/Variables.scss';
+
+$primary-color-lightest: #ffffff;
+$primary-color-lightest-hover: whitesmoke;
+$primary-color-light: #e5e5e5;
+$primary-color-dark: #4a4a4a;
+$accent-color: $calypso;
+
+button.CoveoActionButton.coveo-actionbutton {
+    height: 36px;
+    border: solid 2px $primary-color-light;
+    border-radius: 3px;
+    background-color: $primary-color-lightest;
+    color: $primary-color-dark;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    white-space: nowrap;
+}
+
+.CoveoActionButton {
+    &:not(.coveo-actionbutton-icon) {
+        padding: 0 16px;
+    }
+
+    &.coveo-actionbutton-icon {
+        width: 36px;
+    }
+
+    .coveo-actionbutton_icon {
+        line-height: 0;
+        font-size: 16px;
+
+        svg {
+            height: 1em;
+            width: 1em;
+            fill: $primary-color-dark;
+        }
+
+        + .coveo-actionbutton_title {
```

Comment:
here too... wait is this on purpose?

Reply:
Yes, see the other comment. This is inspired by the [SLDS naming convention](https://www.lightningdesignsystem.com/guidelines/markup-and-style/).